index.html

```diff
@@ -9,21 +9,25 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
-    <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
-      <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
-          <li>
-            <a class="nav-link" href="#contact"
-              >Contact <i class="far fa-envelope"></i
-            ></a>
+    <header class="header">
+      <nav class="navbar">
+        <a href="/" class="nav-logo">Training</a>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#projects" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about-me" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact" class="nav-link">Contact</a>
```

Comment:
- [ ] As the previous reviewer mentioned, kindly follow the design used in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template. 
- [ ] Please align the link elements to the top left part of the screen as shown below 
- [ ] Also it would be nice if you use the Figma font-styling and colors as it helps your project look better and engage anyone who visits it.

> **hint**: Setting `display` of the parent container to `flex` and `justify-content` and `align-items` to `flex-start` can help you achieve the result. Please refer [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for more on positioning elements with flex.

#### FIgma design
![image](https://user-images.githubusercontent.com/61377322/127959175-592567a4-659e-49e7-81d9-31c951fda854.png)

#### Your project
![image](https://user-images.githubusercontent.com/61377322/127959165-4f96327e-bed0-4bc7-ae58-4e81f1eea67b.png)